docs/src/man/getting_started.md

```diff
@@ -558,6 +558,21 @@ Equivalently, the `in` function can be called with a single argument to create
 a function object that tests whether each value belongs to the subset
 (partial application of `in`): `df[in([1, 5, 601]).(df.A), :]`.
 
+!!! note
+
+    While data frames support indexing syntax that is very similar to
+    matrices, subsetting from a `DataFrame` will return a copy, not
+    a view.
+
+    The only subsetting situations where data frames will **not** return a copy are:
+
+    - when a `!` is placed in the first indexing position (`df[!,:A]`, or `df[!, [:A, :B]]`),
+    - when using `.` notation (`df.A`),
+    - when a single row is selected using a single integer (`df[1, :A]`), or
```

Comment:
```suggestion
    - when a single row is selected using an integer (`df[1, :A]`), or
```

Reply:
I feel like there's value in `a single` to emphasize that this doesn't apply for collections of integers (e.g. `df[[1,2], :A]`). You are correct that `an integer` implies just one, but not very strongly.

Reply:
ok - let us wait what to hear native speakers feel :smile:. Thank you for caring for such details.